pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -18,11 +18,11 @@ package dra
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"io"
 	"net"
 
+	"github.com/pkg/errors"
```

Comment:
I'm pretty sure you want the standard `errors` package, right?

Reply:
As far as I remember the linter error was something about wrapping external errors. Does standard `error` package has `Wrap` method?

Reply:
Used %w format specifier instead of errors.Wrap. Seems that this method is more preferred in k8s. Thanks for pointing it out.

Reply:
Not just in Kubernetes. Go itself has embraced error wrapping, which made github.com/pkg/errors pretty much obsolete.